Examples/Swift/RBQFRCSwiftExample/MainTableViewController.swift

```diff
@@ -243,58 +244,53 @@ class MainTableViewController: UITableViewController {
 // MARK: -
 
 extension MainTableViewController: FetchedResultsControllerDelegate {
-    func controllerWillChangeContent<T : Object>(controller: FetchedResultsController<T>) {
+    func controllerWillChangeContent<T : Object>(_ controller: FetchedResultsController<T>) {
         self.tableView.beginUpdates()
     }
-    
-    func controllerDidChangeObject<T : Object>(controller: FetchedResultsController<T>, anObject: SafeObject<T>, indexPath: NSIndexPath?, changeType: NSFetchedResultsChangeType, newIndexPath: NSIndexPath?) {
-        
+
+    func controller<T : Object>(_ controller: FetchedResultsController<T>, didChangeObject anObject: SafeObject<T>, atIndexPath indexPath: IndexPath, forChangeType type: NSFetchedResultsChangeType, newIndexPath: IndexPath) {
```

Comment:
The `indexPath` and `newIndexPath` are optionals in the `NSFetchedResultsController` API so this causes a crash when `nil` is passed through. Should be:

```swift
func controller<T: Object>(_ controller: FetchedResultsController<T>, didChangeObject anObject: SafeObject<T>, atIndexPath indexPath: IndexPath?, forChangeType type: NSFetchedResultsChangeType, newIndexPath: IndexPath?)
```